src/css/profile/mobile/common/popup.less

```diff
@@ -109,7 +109,25 @@
 		}
 
 		.ui-popup-body-checkbox {
-			margin-top: 8px;
+			margin-top: 8 * @px_base;
+
+			.ui-listview.ui-content-area {
+				height: 32 * @px_base;
+
+				.ui-li-has-checkbox {
+					height: 32 * @px_base;
+
+					.ui-li-text {
+						margin: 5 * @px_base 0 0 0;
+						padding: 0;
+
+						.ui-li-text-title {
+							height: 23 * @px_base;
+							line-height: 23 * @px_base;
+						}
+					}
+				}
+			}
```

Comment:
I have a question about dialog.html.
Why do you use the list element only for the checkbox?
``` html
				<div class="ui-popup-content">
					Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
					<div class="ui-popup-body-text ui-popup-body-checkbox">
						<ul class="ui-listview ui-content-area">
							<li class="ui-li-has-checkbox">
								<input checked="checked" class="checkboxes" name="checkbox-1" type="checkbox" />
								<div class="ui-li-text">
									<span class="ui-li-text-title" id="checkbox-text-1">
										Do not show again
									</span>
								</div>
							</li>
						</ul>
					</div>
```

Reply:
@HunseopJeong 
Does the checkbox you commented mean "ui-popup-body-checkbox"?